athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/AwsRestHighLevelClient.java

```diff
@@ -117,19 +152,6 @@ public SearchResponse getDocuments(SearchRequest request)
         return searchHit.getSourceAsMap();
     }
 
-    /**
-     * Shuts down the client.
-     */
-    public void shutdown()
-    {
-        try {
-            close();
-        }
-        catch (IOException error) {
-            logger.error("Unable to shutdown client:", error);
-        }
-    }
```

Comment:
Why are you removing this? Something needs to be able to shut these down eventually even if you are reusing the client.

Reply:
shutdown() was a wrapper around the REST client's close() method. The wrapper was needed because Mockito V1.x could not mock final methods. Since I've upgraded to Mockito V3.x, this method is no longer needed.